src/components/molecules/ZopaFooter/SocialLinks/SocialLinks.tsx

```diff
@@ -13,6 +15,18 @@ export interface ISocialLinkData {
   label: string;
 }
 
+const SFlexRow = styled(FlexRow)`
+  @media (max-width: 768px) {
+    justify-content: center;
+    margin-top: 32px;
+  }
+`;
+
+const SFooterLink = styled(FooterLink)`
```

Comment:
if you need those extra 8px on the side, you can just do `<FlexRow gutter={24}>` (it's `16` by default)